client/components/user/Settings/SetImage.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <v-card class="card-img">
+    <div class="header">
+      <v-avatar :size="options.height">
+        <croppa
+          v-if="isEditing || !currentImage"
+          v-model="croppa"
+          v-bind="options"
+          @file-choose="uploadCroppedImage"
+          placeholder=""
+          prevent-white-space
+          show-loading>
+        </croppa>
+        <img v-if="!isEditing" :src="currentImage">
+        <div>
+          <div v-if="!isEditing && !currentImage" class="croppa-tools">
+            <v-icon class="placeholder-icon">mdi-account</v-icon>
+          </div>
+          <div v-if="!isEditing" class="croppa-tools actions">
+            <v-icon @click="editImage" dark>mdi-pencil</v-icon>
+            <v-icon v-if="currentImage" @click="removeImage" dark>mdi-delete</v-icon>
+          </div>
+        </div>
+      </v-avatar>
+    </div>
+    <v-card-actions class="img-actions">
+      <div v-if="isEditing" class="edit-actions">
+        <v-layout mt-4 row justify-space-around>
+          <v-btn @click="croppa.zoomIn()" color="blue-grey" flat fab>
+            <v-icon>mdi mdi-magnify-plus</v-icon>
+          </v-btn>
+          <v-btn @click="croppa.zoomOut()" color="blue-grey" flat fab>
+            <v-icon>mdi mdi-magnify-minus</v-icon>
+          </v-btn>
+        </v-layout>
+        <v-flex pa-4 d-flex>
+          <v-btn @click="doneEditing" color="light-blue darken-3" flat large>
+            <v-icon dark>mdi-image</v-icon>
+            Save
+          </v-btn>
+          <v-btn @click="cancelEditing" color="blue-grey" flat large>
+            <v-icon dark>mdi-close</v-icon>
+            Cancel
+          </v-btn>
+        </v-flex>
+      </div>
+    </v-card-actions>
+  </v-card>
+</template>
+
+<script>
+import assetsApi from '../../../api/asset';
+import EventBus from 'EventBus';
+import { mapActions, mapGetters } from 'vuex-module';
+
+const appChannel = EventBus.channel('app');
+
+export default {
+  name: 'user-settings',
+  data() {
+    return {
+      isEditing: false,
+      disabled: false,
+      currentImage: null,
+      timeout: 2500,
+      croppa: {}
+    };
+  },
+  computed: {
+    ...mapGetters(['user']),
+    options() {
+      return {
+        accept: 'image/*',
+        width: 200,
+        height: 200,
+        disabled: this.disabled,
+        showRemoveButton: false,
+        initialImage: this.currentImage
+      };
```

Comment:
Same, as above, convert it to lambda. `computed` methods receive `vm` as first param...